### PR TITLE
Khso/GitHub busybox

### PR DESCRIPTION
**Pull request by ksorensen**
Various additions to busybox recipe

### CODE REVIEW

File: recipes/busybox/busybox.inc [hunk 1]
```diff
@@ -43,6 +43,7 @@ SRC_URI += "\
 "
 
 PACKAGES:<USE_busybox_udhcpc = "${PN}-udhcpc-scripts "
+SRC_URI:>USE_busybox_udhcpc = " file://udhcp-simple-script.patch"
```
Comment (kimrhh) [inline recipes/busybox/busybox.inc]: i don't think this falls under "sane defaults", if output like this is a problem it should be worked around downstream. Another possibility is to make it configurable, and not done by default.

File: recipes/busybox/files/busybox-syslogd [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
-	kill `cat $PIDFILE`
-        ;;
+	start)
+		splash_progress PLACEHOLDER
+		syslogd -S $FILE $REMOTE $ROTATE_SIZE $ROTATE_N $LOG_LEVEL
+		;;
+	stop)
+		kill `cat $PIDFILE`
+		;;
 esac
```
Comment (kimrhh) [inline recipes/busybox/files/busybox-syslogd]: could we drop all the whitespace changes?

**Comment by kimrhh** [comment]
other than the two comments, looks good

File: recipes/busybox/files/busybox-syslogd (hunk 2 above)
Comment (ksorensen) [inline recipes/busybox/files/busybox-syslogd]: I have removed the white space changes.

File: recipes/busybox/busybox.inc (hunk 1 above)
Comment (ksorensen) [inline recipes/busybox/busybox.inc]: This commit is removed.

**Pull request merged**